Octokit.Generators/Octokit.Generators.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>A set of code generators for Octokit.NET backed by the GitHub REST API Open API descriptions</Description>
+    <AssemblyTitle>Octokit.Generators</AssemblyTitle>
+    <Authors>GitHub</Authors>
+    <Version>0.0.0-dev</Version>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Why net 5 and not 6? 5 isn’t LTS.

Reply:
I did this primarily because I didn't want to add another SDK runtime to the container (see ci build [output](https://github.com/octokit/octokit.net/runs/7758616506?check_suite_focus=true)).  Looking at it now - that does not seem like a great reason given that I enabled prebuilds on this repo. 

I'll add the SDK and change it.

Reply:
Yeah, I just verified.. adding it in really didn't impact the CI build time. I've added .NET 6.0 to the workflows and changed the project ref.

Reply:
❤️